templates/observability/what-is-observability.html

```diff
@@ -112,26 +112,40 @@ <h3 class="p-heading--4">Distributed traces</h3>
     <div class="row u-equal-height">
         <div class="col-5 u-align--center u-hide--small u-vertically-center">
             {{ image (
-                url="https://assets.ubuntu.com/v1/f8ac79f6-real-user-monitoring.svg",
-                alt="Real user monitoring",
-                width="180",
-                height="150",
+                url="https://assets.ubuntu.com/v1/32c10814-production-profiling.svg",
+                alt="Production profiling",
```

Comment:
Could you leave the alt text for these icons blank, as it adds no value to the adjacent copy. Only adds replication for screen readers.